### ISSUE TITLE
Need tutorial.

### ISSUE THREAD

**Issue by aka-lux** [PatchworkMC/patchwork-patcher#93]
Only a file don't help to solve the problem. Put a tutorial file because I am SURE most of the people don't know how to convert things, like me

**Comment by cittyinthecloud** [comment]
Copied from the Discord #welcome:

- Can I use it?
The short answer is: **No.**
The long answer:
- Patchwork only runs a few mods (as of writing only Pam's Foodcore)
- Patchwork is not looking for testers, and we do not want people who are not writing code for Patchwork using the software.
- As such, nobody is required to help you run Patcher or use Patchwork API with your favorite mods.
**With that being said,**
- You can always open bug reports on our github pages
- Everyone is encouraged to ask questions in the Development category about writing and using Patchwork's code, or asking about how Patchwork works.
- This note isn't a restriction on giving help, but instead a warning that you shouldn't expect official support.
TL;DR: **We are not required to help you run Patchwork, and you don't want to try**
